contentcuration/contentcuration/models.py

```diff
@@ -262,6 +262,8 @@ class ContentNode(MPTTModel, models.Model):
     # TODO: disallow nulls once existing models have been set
     original_channel_id = UUIDField(primary_key=False, editable=False, null=True) # Original channel copied from
     source_channel_id = UUIDField(primary_key=False, editable=False, null=True) # Immediate channel copied from
+    original_source_node_id = UUIDField(primary_key=False, editable=False, null=True) # Original node_id of node copied from (TODO: original_node_id clashes with original_node field - temporary)
```

Comment:
Oh, interesting, yeah. I guess just add a comment to rename this field after we remove original_node.